articles/article-117-Why-stick-to-one-example.md

```diff
@@ -0,0 +1,35 @@
+<!-- BEGIN TITLE -->
+# Why stick to one example
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Using the same example, again and again, seems to be quite boring without any added benefit.
+In some instances, however, this can be used for some positive effect. Using a known example to show different things (like methods or algorithms) saves time, as the people don't have to think about the example used in much detail.
+In this book, the common example is the Fibonacci Sequence which starts with 0 and 1. Every next number is the sum of their two predecessors. 
+
+0, 1, 1, 2, 3, 5, 8, 13, 21,... 
+
+The well-known picture according to this Sequence is the Fibonacci spiral.
```

Comment:
A well-known picture using this sequence is the Fibonacci spiral.

Reply:
Done.